components/textfield/metadata/textfield.yml

```diff
@@ -25,6 +40,34 @@ sections:
       * `.spectrum-Textfield.is-valid` - when the input has an explicit valid state
       * `.spectrum-Textfield.is-invalid` - when the input has an explicit invalid state
 
+      #### Valid Icon
+      Small
```

Comment:
yeah, those mare a bit different for Express. see my previous comment about #1542